src/views/homepage/components/certifications-section/components/split-layout-with-background/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactNode } from 'react'
+import classNames from 'classnames'
+import s from './split-layout-with-background.module.css'
+
+/**
+ * Render a split layout with a background.
+ */
+export function SplitLayoutWithBackground({
```

Comment:
Split this out for clarity. Not sure it's worth abstracting further, since different contexts will probably have different needs for column widths, but figured why not split it out given it's relatively generic.

Reply:
I like it! seems like a fairly common pattern with these 'marketing-y' pages. Could be worth lifting up to root `/components` if the intent is for other views to be able to use?

Reply:
I dont mind if we leave it here though, its up to you 👍🏻

Reply:
Sweet, moved up to `components` - and added a note that the layout isn't exactly 50/50 just in case 👍